pelican_bib/bib.py

```diff
@@ -160,6 +201,159 @@ def add_publications(generator):
     generator.context['publications_lists'] = publications_lists
 
 
+class Bibliography(Directive):
+    """
+    Directive to embed bibliographies into articles/posts.
+
+    Usage::
+
+        .. bibliography:: PUBLICATIONS_SRC
+          :template: TEMPLATE_NAME (optional)
+          :options: TEMPLATE_OPTIONS_DICT (optional)
+          :class: CLASS_NAME (optional)
+          :filter_tag: FILTER_TAG (optional)
+          :pybtex_style_args: PYBTEX_STYLE_ARGS (optional)
```

Comment:
Sounds good. The other `pybtex_` options are:
- `sorting_style`
- `abbreviate_names`
- `name_style`

Reply:
Hm... There is already the config paramter named `PUBLICATIONS_CUSTOM_STYLE`. In my opinion `PUBLICATIONS_STYLE_ARGS` should be a good name for the args of the default or custom style. The name `PUBLICATIONS_PYBTEX_STYLE_ARGS` looks a bit odd.  

So I would rather remove the prefix "pybtex" and simply refer to the respective [instance of pybtex.style.formatting.BaseStyle] as "style".

Using the `style_` prefix consequently, it would also looks a bit weird:
```
:style_args: {'sorting_style': 'author_year_title',
              'abbreviate_names': False,
              'name_style': 'lastfirst' }
:style_sorting_style: author_year_title
:style_abbreviate_names: False
:style_name_style: lastfirst
```

What do you think?